src/OrbitQt/TimeGraphLayoutWidget.cpp

```diff
@@ -32,14 +32,26 @@ TimeGraphLayoutWidget::TimeGraphLayoutWidget(QWidget* parent)
   AddWidgetForProperty(&rounding_radius_);
   AddWidgetForProperty(&rounding_num_sides_);
   AddWidgetForProperty(&text_offset_);
-  AddWidgetForProperty(&left_margin_);
```

Comment:
You should also erase the widget for track_tab_width_, track_tab_height_ and track_tab_offset_, or I am missing something?

Reply:
good point, done, thanks.